File: CSharp/SpaceVIL/SpaceVIL/Cores/TextPrinter.cs
using System;
using System.Drawing;
using System.IO;

namespace SpaceVIL
{
    internal class TextPrinter
    {
        public byte[] Texture = null;
        public int XTextureShift = 0;
        public int YTextureShift = 0;
        public int WidthTexture = 0;
        public int HeightTexture = 0;
        //public Color foreground = Color.Black;

        public TextPrinter()
        {
            Texture = null;
        }

        internal TextPrinter(byte[] bb)
        {
            Texture = bb;
        }
        /*
        public Color GetForeground() {
            return foreground;
        }
        */
    }
}

File: Java/SpaceVIL/spacevil/src/main/java/com/spvessel/TextPrinter.java
package com.spvessel;

import java.nio.ByteBuffer;

public class TextPrinter {
    ByteBuffer texture = null;
    int xTextureShift = 0;
    int yTextureShift = 0;
    int widthTexture = 0;
    int heightTexture = 0;

    TextPrinter() {
        texture = null;
    }

    TextPrinter(ByteBuffer bb) {
        texture = bb;
    }
}